pkgs/games/rare/default.nix

```diff
@@ -1,15 +1,15 @@
 { lib, fetchFromGitHub, buildPythonApplication, qt5
-, psutil, pypresence, pyqt5, python, qtawesome, requests }:
+, legendary-gl, pypresence, pyqt5, python, qtawesome, requests, typing-extensions }:
 
 buildPythonApplication rec {
   pname = "rare";
-  version = "1.8.9";
+  version = "1.9.0";
 
   src = fetchFromGitHub {
     owner = "Dummerle";
     repo = "Rare";
     rev = version;
-    sha256 = "sha256-2l8Id+bA5Ugb8+3ioiZ78dUtDusU8cvZEAMhmYBcJFc=";
+    sha256 = "sha256-UN7WJM2r45iyqnn7iV2CmPZUfb1ulzkJhjkIbstwTe4=";
     fetchSubmodules = true;
```

Comment:
```suggestion
```
As `legendary-gl` is now being built as a Python dependency instead of a submodule, you can now remove this